README.md

```diff
@@ -1,13 +1,30 @@
-# meteodata-lab
-<!-- To update when it's released -->
-[![PyPI version](https://img.shields.io/pypi/v/your-package-name.svg)](https://pypi.org/project/your-package-name/)
-
-A model data processing framework based on xarray.
-
-**DISCLAIMER**
+<p align="center">
+  <picture>
+    <img src="meteodata_lab_logo_with_name-optimized.gif" height="160">
+  </picture>
+</p>
+
+<p align="center">
+    <a href="https://pypi.org/project/meteodata-lab/">
+    <img src="https://img.shields.io/pypi/v/meteodata-lab.svg?color=ff69b4" alt="PyPI version">
+    </a>
+    <a href="https://github.com/meteoswiss/meteodata-lab/releases">
+    <img src="https://img.shields.io/github/v/release/meteoswiss/meteodata-lab?color=purple&label=Release" alt="Latest Release">
+    </a>
+    <a href="https://opensource.org/licenses/mit">
+    <img src="https://img.shields.io/badge/licence-MIT-blue.svg" alt="Licence">
+    </a>
+</p>
+
+<p align="center">
+    <a href="#installation">Installation</a> •
+    <a href="https://meteoswiss.github.io/meteodata-lab/">Documentation</a>
+</p>
 
 > [!WARNING]
-> This project is BETA and will be experimental for the forseable future. Interfaces and functionality are likely to change, and the project itself may be scrapped. Do not use this software in any project/software that is operational.
+> This project is in BETA and under active development. Interfaces and functionality are subject to change.
+
+Meteodata-lab is a NumPy/Xarray-based Python library for processing and analyzing gridded meteorological data. It supports GRIB (read/write) and is tailored to common workflows that require data interpolation, regridding to custom grids (e.g., Swiss grid or rotated lat/lon), and the computation of advanced meteorological fields.
```

Comment:
Perhaps there could also be a sentence which conveys the fact that meteodata-lab provides operators which maintain the consistency of the GRIB metadata (which is why it is able to write back to GRIB)